src/User/ProfilePage.jsx

```diff
@@ -26,7 +60,21 @@ const ProfilePage = ({ user, isLoading, error }) => {
         <p className="email">Email: {email}</p>
         <p className="occupation">Occupation: {occupation}</p>
       </StyledCard>
-    </StyledForm>
+      <StyledCard>
+        <StyledCardHeading className="favourite-conferences">
+          Favourited conferences
+        </StyledCardHeading>
+        <StyledFavouritesList>
```

Comment:
I think it would be nice to pull this list out into its own component, helps seperate the logic